api/config.go

```diff
@@ -56,9 +71,13 @@ Failure Threshold: {{.FailureThreshold}}
 Exec Probe: {{or .Exec "N/A"}}
 HTTP GET Probe: {{or .HTTPGet "N/A"}}
 TCP Socket Probe: {{or .TCPSocket "N/A"}}`)
-	if err != nil { panic(err) }
+	if err != nil {
+		panic(err)
```

Comment:
I think it's fine to panic if the SDK cannot compile the template, rather I think this is a valid situation where a panic is warranted. There is no way someone could recover from this, and a `String()` method returns no errors, anyways.